src/Events/Services/AppSubscriptionService.cs

```diff
@@ -137,16 +133,26 @@ private async Task<string> GetPartyFromAlternativeSubject(string alternativeSubj
         {
             int partyId = 0;
 
-            if (alternativeSubject.StartsWith(OrganisationPrefix))
+            /* Both OrgPrefix and OrganisationPrefix is assumed to give us the organisation number of an 
+             * organisation. OrgPrefix is NOT for the application owner acronym like it is many other 
+             * places. OrgPrefix was kept as is when adding OrganisationPrefix for backwards compability.
+             */
+
+            if (alternativeSubject.StartsWith(OrgPrefix))
```

Comment:
Can we consider handling the orgPrefix and OrganisationPrefix in the same if-clause to better demonstrate that they represent the same "thing" ?

Reply:
Alternatively, we could have the controller be responsible for mapping the alternative subject to a format that the service can handle. E.g. mapping all /org/ to /organisation/. 
Limiting the backward compatibility to the controller and making it clear there that use of /org/ is deprecated?

Reply:
I like the code flow the way it is, but I moved the to cases closer together and improved the explanation.